osquery/core/sql/query_performance.h

```diff
@@ -34,6 +34,9 @@ struct QueryPerformance {
 
   /// Average memory differentials. This should be near 0.
   unsigned long long int average_memory{0};
+
+  /// Total bytes for the query
+  unsigned long long int output_size{0};
```

Comment:
The comment on `average_memory` implies it's the memory used by the differentials, though I always thought it was the memory used by osquery?

Reply:
Shameless plug: I have changed that comment and the column description here https://github.com/osquery/osquery/pull/7438

Still have to open a Blueprint but it seems to be that that counter is not so useful; what the user is most likely interested in is the peak memory usage, which has to be gathered by continuosly sampling memory usage during the query execution.